### DISCUSSION TITLE
Ctrl + Alt + Backspace = evade the blocking made by the tutor closing the session

### DISCUSSION THREAD

**Discussion by GerardoSystem0o0** [epoptes/epoptes#77] [Q&A]
When the student has the session open and the tutor blocks the screen with epoptes
the keyboard continues to work, allowing you to close the session by pressing ctrl + alt + backspace (<----)

**Comment by GerardoSystem0o0** [comment]
the problem is persistent in the new version of epoptes

**Comment by alkisg** [comment]
Linux Mint configures "Ctrl+Alt+Backspace" to reset Xorg?
You can disable it from your keyboard-configuration package options.
Normal Ubuntu doesn't default to that.

System keys like Ctrl+Alt+Backspace or Alt+Ctrl+Delete shouldn't be intercepted by epoptes-client...
